.github/workflows/ci.yml

```diff
@@ -1,70 +1,40 @@
 name: ci
 
 on:
-- pull_request
-- push
+  - pull_request
+  - push
 
 jobs:
   test:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     strategy:
       matrix:
-        name:
-        - Node.js 0.10
-        - Node.js 0.12
-        - io.js 1.x
-        - io.js 2.x
-        - io.js 3.x
-        - Node.js 4.x
-        - Node.js 6.x
-        - Node.js 8.x
-        - Node.js 10.x
-        - Node.js 11.x
-        - Node.js 12.x
-        - Node.js 13.x
-        - Node.js 14.x
-        - Node.js 15.x
-        - Node.js 16.x
-        - Node.js 17.x
-        - Node.js 18.x
-        - Node.js 19.x
-        - Node.js 20.x
-        - Node.js 21.x
-        - Node.js 22.x
+        node-version:
+          - 18.x
+          - 19.x
+          - 20.x
+          - 21.x
+          - 22.x
 
     steps:
-    - uses: actions/checkout@v4
+      - name: Checkout code
+        uses: actions/checkout@v3
```

Comment:
Is there a reason not to use checkout@v4? (https://github.com/actions/checkout/releases/tag/v4.2.1)

Reply:
Added in d87694a. Thanks @inigomarquinez